<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/find.png</normaloff>img/find.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="enter_a_word">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>671</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter a word</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>711</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="to_add_word">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string> ADD NEW WORD</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/plus_in_circle.png</normaloff>img/plus_in_circle.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to_change_word">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>510</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string> CHANGE THE WORD INFORM</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/update.png</normaloff>img/update.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="search_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/find.png</normaloff>img/find.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
